jp/components/grids_and_lists.md

```diff
@@ -20,176 +20,176 @@ _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Nati
+      <a href="grid_virtualization.md" class="feature__more-details">仮想ディレクティブ チュートリアル</a>
     </div>
     <div class="feature__details">
-      <h3 id="virtualization-and-performance">[Virtualization and Performance](grid_virtualization.md)</h3>
-      <p>The updated Ignite UI for Angular Virtualization directive ensures that you can seamlessly scroll through large-scale data sets with millions of points, giving our fastest Angular Data Grid high-speed rendering and no-lag loading.</p>
+      <h3 id="virtualization-and-performance">[仮想化とパフォーマンス](grid_virtualization.md)</h3>
+      <p>Ignite UI for Angular Virtualization ディレクティブは、数百のポイントがある大きなデータセットをシームレスにスクロール、高速な Angular Data Grid の描画や遅延なしの読み込みが可能です。</p>
     </div>
   </div>
 
   <div class="feature">
     <div class="feature__image feature__image--left">
-      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-480.gif 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-1100.gif 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="Paging component allows for programmatic pagination">
-      <a href="grid.md" class="feature__more-details">Paging Component Documentation</a>
+      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-480.gif 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-1100.gif 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="ページング コンポーネントは、コードで改ページを実装できます。">
+      <a href="grid.md" class="feature__more-details">ページング コンポーネント ドキュメント</a>
     </div>
     <div class="feature__details">
       <h3 id="paging">[Paging](grid.md)</h3>
-      <p>Controlling whether a feature is enabled, the paging component allows for templated and programmatic pagination.</p>
+      <p>機能を有効にするかどうかを制御し、ページング コンポーネントはテンプレートやコードによる改ページが可能となります。</p>
```

Comment:
The English here doesn't make sense.  There is no reason to say "Controlling whether a feature is enabled".  Of course our features are optional.  I'd just leave that out and start with "the paging component...".